sdk/entra/Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents/src/Framework/AuthenticationEventData.cs

```diff
@@ -13,17 +14,20 @@ public abstract class AuthenticationEventData
         /// <summary>Gets the event identifier.</summary>
         /// <value>The event identifier.</value>
         [JsonPropertyName("tenantId")]
+        [JsonRequired]
```

Comment:
Gotcha, I agree with both ways. I worry as we move to new version of the Json library, are we going to keep the validation attributes? They seem like overhead we could get rid of. Thoughts? 

I will put in [Required] as it will cause "Validation" exceptions.